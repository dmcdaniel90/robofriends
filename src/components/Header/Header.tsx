import { memo } from "react";

const Header = memo(function Header(): React.JSX.Element {
  return (
    <h1 className="f1">RoboFriends</h1>
  )
})

export default Header;