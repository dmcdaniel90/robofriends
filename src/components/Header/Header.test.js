import { render } from '@testing-library/react';
import Header from './Header';

describe('Header', () => {
  it('expect to render Header component', () => {
   
    const {container} = render(<Header />)
    expect(container.firstChild).toMatchSnapshot();
  })
})