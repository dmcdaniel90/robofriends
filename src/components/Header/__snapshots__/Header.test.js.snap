// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header expect to render Header component 1`] = `
<h1
  class="f1"
>
  RoboFriends
</h1>
`;
